contracts/nois-drand/src/state.rs

```diff
@@ -75,6 +93,12 @@ pub const SUBMISSIONS: Map<(u64, &Addr), StoredSubmission> = Map::new("submissio
 /// The number of submissions done for each round
 pub const SUBMISSIONS_COUNT: Map<u64, u16> = Map::new("counts");
 
+/// Dummy value. Don't rely on the value but just check existence.
+pub const INCENTIVIZED_BY_GATEWAY_MARKER: u8 = 1;
+
+/// A map from round number to marker value
+pub const INCENTIVIZED_BY_GATEWAY: Map<u64, u8> = Map::new(TopKey::Incentivised.as_str());
```

Comment:
why not Map<u64, ()> ?

Reply:
`()` is serialized to `null` in JSON, which is 4 bytes instead of 1

Reply:
haha of course. thanks.